MH4U Dex/CoreDataController.m

```diff
@@ -363,6 +438,87 @@ - (NSUInteger)countForEntityWithEntityName:(NSString *)entityName withPredicate:
     }
 }
 
+- (QuestDrop *)questDropFromDictionary:(NSDictionary *)dropDict
+{
+    NSPredicate *questPredicate = [NSPredicate predicateWithFormat:@"%K == %@", QuestAttributes.id, dropDict[QuestDropQuestIDKey]];
+    if (![self getUniqueEntityWithEntityName:[Quest entityName] withPredicate:questPredicate]) {
```

Comment:
I know I'm late to the game on this project. Please forgive these points:
1) Generally "get" as a prefix is verboten. I'd prefer "fetch" or no prefix for a method like this.
2) Why not use Vokoder? I respect the experienced gained building your own core data stack, but I'd also like you to be up to speed on our standard tools.